sql-statements/sql-statement-flashback-to-timestamp.md

```diff
@@ -0,0 +1,102 @@
+---
+title: FLASHBACK [CLUSTER | DATABASE | TABLE] TO TIMESTAMP
+---
+
+# FLASHBACK [CLUSTER | DATABASE | TABLE] TO TIMESTAMP
+
+TiDB v6.4.0 引入了 `FLASHBACK [CLUSTER | DATABASE | TABLE] TO TIMESTAMP` 语法，其功能是将集群的数据恢复到特定的时间点。
+
+## 语法
+
+```sql
+FLASHBACK TABLE tbl1, tbl2 TO TIMESATMP '2022-09-21 16:02:50';
+FLASHBACK DATABASE db1, db2 TO TIMESATMP '2022-09-21 16:02:50';
+FLASHBACK CLUSTER TO TIMESATMP '2022-09-21 16:02:50';
+```
+
+### 语法图
+
+```ebnf+diagram
+FlashbackToTimestampStmt ::=
+    "FLASHBACK" ("TABLE" TableNameList | "DATABASE" DBNameList | "CLUSTER") "TO" "TIMESTAMP" stringLit
+```
```

Comment:
````suggestion
```ebnf+diagram
FlashbackToTimestampStmt ::=
    "FLASHBACK" ("CLUSTER" | "DATABASE" DBNameList | "TABLE" TableNameList) "TO" "TIMESTAMP" stringLit
```
````
<img width="773" alt="image" src="https://user-images.githubusercontent.com/60599231/195322245-372b2803-fe24-44f9-8f2c-a0b8f39d1845.png">